contributing/code_of_conduct/reporting_guidelines.rst

```diff
@@ -0,0 +1,98 @@
+====================
+
+If you believe someone is violating the code of conduct we ask that you report
+it to the :doc:`enforcement team </contributing/code_of_conduct/enforcement_team>`
+by emailing, twitter, in person or any way you see fit.
+
+**All reports will be kept confidential.** In some cases we may determine that
+a public statement will need to be made. If that's the case, the identities of
+all victims, reporters, and the accused will remain confidential unless those
+individuals instruct us otherwise.
+
+If you believe anyone is in physical danger, please notify appropriate emergency
+services first by calling the relevant local authorities. If you are unsure what
+service or agency is appropriate to contact, include this in your report and we
+will attempt to notify them.
+
+In your report please include:
+
+  * Your contact info for follow-up contact.
+  * Names (legal, nicknames, or pseudonyms) of any individuals involved.
+  * If there were other witnesses besides you, please try to include them as well.
+  * When and where the incident occurred. Please be as specific as possible.
+  * Your account of what occurred.
+  * If there is a publicly available record (e.g. a mailing list archive or a
+    public IRC or Slack log), please include a link.
+  * Any extra context you believe existed for the incident.
+  * If you believe this incident is ongoing.
+  * Any other information you believe we should have.
+
+What happens after you file a report?
+-------------------------------------
+
+You will receive a reply from the :doc:`enforcement team </contributing/code_of_conduct/enforcement_team>`
+acknowledging receipt as soon as possible, but within 24 hours.
+
+The team will immediately meet to review the incident and determine:
+
+  * What happened.
+  * Whether this event constitutes a code of conduct violation.
+  * What kind of response is appropriate.
+
+If this is determined to be an ongoing incident or a threat to physical safety,
+the team's immediate priority will be to protect everyone involved. This means
+we may delay an "official" response until we believe that the situation has ended
+and that everyone is physically safe.
+
+Once the team has a complete account of the events, they will make a decision as
+to how to respond. Responses may include:
+
+  * Nothing (if we determine no code of conduct violation occurred).
+  * A private reprimand from the Code of Conduct response team to the individual(s)
+    involved.
+  * A public reprimand.
+  * An imposed vacation (i.e. asking someone to "take a week off" from a mailing
+    list or Slack).
+  * A permanent or temporary ban from some or all Symfony conference/community
+    spaces (events, meetings, mailing lists, IRC, Slack, etc.)
+  * A request to engage in mediation and/or an accountability plan.
+  * On a case by case basis, the enforcement team might share information an incidents
+    with employers or other 3rd parties in order to fulfill their mandate.
```

Comment:
`On a case by case basis, the enforcement team might share information an incidents with employers or other 3rd parties in order to fulfill their mandate.`

This sounds like something that should only be done in extreme cases. I approve that it could be necessary if someone becomes very destructive, but it should be clear for all that this route should only be taken if there is really no alternative.

Reply:
yeah .. indeed, the emphasis here is on `order to fulfill their mandate`. ie. its not the job of the enforcement team to check if their employees are doing good marketing or even fixing the HR issues if companies.

Reply:
I agree with @hipio on this one. Maybe there could be some clarification as to why it would be necessary to tell an employer about something that might not even be related to the job or the company at all? I see a lot of potential for abuse. Let's for a moment assume that the whole Drupal drama a while back (just citing the most recent example I can recall) caused somebody to lose their jobs because a person decided that this was worth reporting to their employer? Is this something the enforcement team would be comfortable with?

Reply:
noting again `order to fulfill their mandate` .. is the mandate maybe not clear enough for you to be comfortable with this section?

I work at Liip and lots of Symfony community members work at Liip. Lets say someone from Liip repeatedly misbehaves quite severely at Symfony events. Maybe even so badly that they decide that person needs to be banned from future Symfony events. In that situation the enforcement team might decide that in order to protect the Symfony developers working at Liip need to be protected by informing Liip. This is something that would be considered "fulfilling their mandate".

Reply:
Ah, I see. Again, maybe it's the lack of examples that might cause confusion. With your explanation it's clear what you mean by `fulfill their mandate`, thank you.

Now, I'm not sure if it's necessary to include such a specific example in the actual document or if it might be enough to clarify this when the question arises. For me, the explanation eliminates my concerns about the section. 👍

Reply:
Yeah .. its a bit tricky .. we can and should not try to cover every eventuality. BTW we are also planning to add a document of real world issues (mostly taken from the php women experience for now) which will be linked from the CoC.

Also as noted here https://github.com/symfony/symfony-docs/pull/9340#issuecomment-368479367 .. it might be a good idea that we clarify our fundamental aims that overarch the entire process.

Reply:
I've updated this. Thank you for the feedback.